frontend/src/component/Workspace/OpenContentApp.jsx

```diff
@@ -22,26 +22,28 @@ export class OpenContentApp extends React.Component {
     } = this.props
 
     if (isNaN(workspaceId) || workspaceId === -1) return
+    const type = match.params.type
 
-    if (['type', 'idcts'].every(p => p in match.params) && match.params.type !== 'contents') {
-      if (isNaN(match.params.idcts) || !contentType.map(c => c.slug).includes(match.params.type)) return
+    if (type !== 'contents' && ['type', 'idcts'].every(p => Object.prototype.hasOwnProperty.call(match.params, p))) {
```

Comment:
what's the difference between "in" operator and ".hasOwnProperty.call()" ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

> The `in` operator returns true for properties in the prototype chain. (If you want to check for only non-inherited properties, use Object.prototype.hasOwnProperty() instead.)

`in` has often unattended side effects and should generally be avoided. See https://2ality.com/2012/01/objects-as-maps.html